tutorials/tutorial-hello-timescale.md

```diff
@@ -0,0 +1,803 @@
+
+If you’ve made it this far, you’ve successfully completed Mission 2 and now 
+have a basic understanding of how to analyze time-series data using TimescaleDB!
+
+### Mission 3: Monitoring
+
+We can also use the time-series data from taxi rides for monitoring 
+a ride’s current status.
+
+>:WARNING: A more realistic setup would involve creating a data pipeline that streams sensor data directly from the cars into TimescaleDB. However, we will use the January 2016 data to illustrate the underlying principles that are applicable regardless of setup.
+
+#### How many rides took place every 5 minutes for the first day of 2016?
+It’s January 1st 2016. NYC riders have celebrated New Year’s Eve, and using taxi 
+cabs to travel to and from their first gathering of the new year.
+
+The first thing you may like to know is how many rides have recently taken 
+place. We can approximate that by counting the number of rides that were 
+completed on the first day of 2016, in 5 minute intervals.
+
+While it's easy to count how many rides took place, there is no easy way 
+to segment data by 5 minute time intervals in PostgreSQL. As a result, we 
+will need to use a query similar to the query below: 
+
+```sql
+-- Vanilla Postgres query for num rides every 5 minutes
+SELECT
+  EXTRACT(hour from pickup_datetime) as hours,
+  trunc(EXTRACT(minute from pickup_datetime) / 5)*5 AS five_mins,
+  COUNT(*)
+FROM rides
+WHERE pickup_datetime < '2016-01-02 00:00'
+GROUP BY hours, five_mins;
+```
+
+It may not be immediately clear why the above query returns rides segmented 
+by 5 minute buckets, so let’s examine it more closely, using the sample time 
+of 08:49:00.
+
+In the above query, we first extract the hour that a ride took place in:
+
+```sql
+EXTRACT(hour from pickup_datetime) as hours
+```
+  
+So for 08:49 our result for `hours` would be 8. Then we need to calculate, `five_mins`, 
+the closest multiple of 5 minutes for a given timestamp. To do this, we calculate 
+the quotient of the minute that a ride took place divided by 5. Then we truncate 
+the result to take the floor of that quotient. Afterward, we multiply that truncated 
+quotient by 5 to, in essence, find the 5 minute bucket that the minute is closest to:
+
+```sql
+trunc(EXTRACT(minute from pickup_datetime) / 5)*5 AS five_mins
+```
+
+So our result for time of 08:49 would be `trunc(49/5)*5 = trunc(9.8)*5 = 9*5 = 45`, 
+so this time would be in the 45min bucket. After extracting both the hours and which 
+5 minute interval the time fell into, we’d then group our results, first by the 
+`hours` and then the `five_mins` interval. Whew, that was a lot for a conceptually 
+simple question!
+
+>:TIP: When time is the primary axis on which you're examining data, ordering and grouping information by time is critical. Vanilla PostgreSQL is often not good enough for this purpose. This is one of the biggest reasons why people choose purpose-built time-series databases like TimescaleDB.
```

Comment:
This whole tip on further reflection seems salesy?  Especially since the point is made in the paragraph below